test/plugin_functional/plugins/rendering_plugin/server/plugin.ts

```diff
@@ -45,14 +43,11 @@ export class RenderingPlugin implements Plugin {
         const { includeUserSettings } = req.query;
         const app = { getId: () => id! };
         const options: Partial<IRenderOptions> = { app, includeUserSettings };
-        const body = await context.core.rendering.render(options);
 
-        return res.ok({
-          body,
-          headers: {
-            'content-security-policy': core.http.csp.header,
-          },
-        });
+        if (includeUserSettings) {
+          return res.renderCoreApp(options as any);
+        }
+        return res.renderAnonymousCoreApp(options as any);
```

Comment:
Two questions:
- Is this `app` property actually used anymore? It appears that the toolkit does not pass this to RenderingService
- Why do we need to cast this to `any`?

Reply:
> Is this app property actually used anymore? It appears that the toolkit does not pass this to RenderingService

Good catch. `App` should be passed to keep the LP rendering working, but I removed it during cleanup. https://github.com/elastic/kibana/blob/e6c23ea9b2913cafd6e3aae528d4be78bcb7850d/src/legacy/ui/ui_render/ui_render_mixin.js#L175-L197

> Why do we need to cast this to any?

NP typings do not support `app` & `vars` passed for `rendering service` https://github.com/elastic/kibana/blob/470056464703d3993b69fd726f3e152320eae126/src/core/server/rendering/types.ts#L95-L102